src/style.scss

```diff
@@ -153,8 +153,10 @@ label {
 input,
 .input {
   border: 2px solid $color-primary;
+  border-right: 0;
   border-radius: 0.3em;
-  padding: 0.5em 1em;
+  background-color: $background-color;
+  padding: 0.5em 1em 0.5em;
   display: flex;
   flex-grow: 1;
```

Comment:
Have you check if these changes affect the rest of the addon? Because you're changing the general `input` style, which is used in some other places. Make sure this does not break the rest. Keep in mind that not all inputs have a select/button on the right